### ISSUE TITLE
where can I find 'enwiki-20171001-pages-meta-current-withlinks-abstracts'

### ISSUE THREAD

**Issue by slobLX** [THUDM/CogQA#14]
Hello, I only find four data files in the link of hotpotQA you give:
Training set, Dev set(distractor), Dev set and Test set

So how can I get 'enwiki-20171001-pages-meta-current-withlinks-abstracts'

Thank you!

**Comment by q82219654** [comment]
https://nlp.stanford.edu/projects/hotpotqa/enwiki-20171001-pages-meta-current-withlinks-abstracts.tar.bz2
